File: golang/src/p6/main.go
package main

import "fmt"

func main() {
	sum := 0
	sum_of_squares := 0
	for i := 1; i <= 100; i++ {
		sum += i
		sum_of_squares += i * i
	}

	square_of_sum := sum * sum
	fmt.Println(square_of_sum - sum_of_squares)
}

File: cpp/src/p6/main.cc
#include <iostream>

int main() {
	int sum = 0;
	int sum_of_squares = 0;
	for (int i = 1; i <= 100; ++i) {
		sum += i;
		sum_of_squares += i * i;
	}

	const int square_of_sum = sum * sum;
	std::cout << (square_of_sum - sum_of_squares) << "\n";
}
